core/iwasm/include/gc_export.h

```diff
@@ -7,6 +7,7 @@
 #define _GC_EXPORT_H
 
 #include "wasm_export.h"
+#include <stdbool.h>
```

Comment:
Seems no need to change the return type of wasm_ref_type_set_type_idx, and no need to include stdbool.h?